src/crl.c

```diff
@@ -141,6 +158,11 @@ static int InitCRL_Entry(CRL_Entry* crle, DecodedCRL* dcrl, const byte* buff,
 /* Free all CRL Entry resources */
 static void FreeCRL_Entry(CRL_Entry* crle, void* heap)
 {
+#ifdef CRL_USE_CERT_BUFFERS
+    if (crle != NULL && crle->certs != NULL) {
+        XMEMSET(crle->certs, 0, crle->totalCerts*sizeof(RevokedCert));
```

Comment:
If using `crle->totalCerts` here then you need to make sure it is not `> CRL_MAX_REVOKED_CERTS`. Any reason to not just use `CRL_MAX_REVOKED_CERTS` here?